db/migrate/20190412163011_remove_user_joined_post_table.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveUserJoinedPostTable < ActiveRecord::Migration
+  def change
+    drop_table :user_joined_post
```

Comment:
In `production` this table contains just one record, I would say it was just a test.